website/docs/.vitepress/theme/Sandbox.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :id="playgroundId" :class="{ 'playground-runtime': !hideCode, 'documentation-sandbox': true }" role="region" aria-label="Playground">
+  <div :id="playgroundId" :class="{ 'playground-runtime': !hideCode, 'documentation-sandbox': true }" aria-label="Playground">
```

Comment:
looks like you should add a hideCode condition

Reply:
Found it, thanks. Fixed.